backend/src/LeanCode.AppRating/Processes/SendEmailOnLowRateSubmitted.cs

```diff
@@ -0,0 +1,46 @@
+using LeanCode.AppRating.Configuration;
+using LeanCode.AppRating.EmailViewModels;
+using LeanCode.SendGrid;
+using MassTransit;
+using SendGrid.Helpers.Mail;
+using Serilog;
+
+namespace LeanCode.AppRating.Processes;
+
+public class SendEmailOnLowRateSubmitted<TUserId> : IConsumer<RateSubmitted<TUserId>>
```

Comment:
Why aren't we putting it near the `SubmitAppRatingCH`?

Reply:
We didn't conclude this discussion last time, do we want to start it once more on weekly meeting?

Reply:
I have too much "free" time I think